src/_gettsim/ids.py

```diff
@@ -31,7 +32,7 @@ def ehe_id(
         + np.minimum(p_id, p_id_ehepartner_or_own_p_id) * n
     )
 
-    return _reorder_ids(result)
+    return reorder_ids(result)
```

Comment:
This will be easy to forget. Could we maybe run this automatically via the `@group_creation_function` decorator?